docs/dev/python/table-api-users-guide/udfs/vectorized_python_udfs.md

```diff
@@ -66,6 +66,7 @@ table_env.sql_query("SELECT add(bigint, bigint) FROM MyTable")
 ## Vectorized Aggregate Functions
 
 Vectorized Python aggregate functions takes one or more `pandas.Series` as the inputs and return one scalar value as output.
```

Comment:
Should add one empty line before "Note". Otherwise, it will appear in the same line of the previous section.